rpc/dial.go

```diff
@@ -294,9 +294,9 @@ func dial(
 	return conn, cached, nil
 }
 
-func lookupMDNSCandidate(ctx context.Context, address string, logger golog.Logger) (*zeroconf.ServiceEntry, error) {
+func lookupMDNSCandidate(ctx context.Context, address string, logger utils.ZapCompatibleLogger) (*zeroconf.ServiceEntry, error) {
 	candidates := []string{address, strings.ReplaceAll(address, ".", "-")}
-	resolver, err := zeroconf.NewResolver(logger, zeroconf.SelectIPRecordType(zeroconf.IPv4))
+	resolver, err := zeroconf.NewResolver(logger.(*zap.SugaredLogger), zeroconf.SelectIPRecordType(zeroconf.IPv4))
```

Comment:
this is just a raw type conversion for now since we aren't passing in RDK loggers to goutils yet, but it seems like we will end up needing some kind of zap conversion to be able to pass loggers into `zeroconf` functions